homeassistant/components/asuswrt/__init__.py

```diff
@@ -70,7 +70,7 @@
 
     await api.connection.async_connect()
     if not api.is_connected:
-        _LOGGER.error("Unable to setup asuswrt component")
+        _LOGGER.error("Unable to setup ASUSWRT component")
```

Comment:
[Style guidelines](https://developers.home-assistant.io/docs/en/development_guidelines.html#log-messages):

> There is no need to add the platform or component name to the log messages. This will be added automatically.

```suggestion
        _LOGGER.error("Unable to setup component")
```